src/IO/ADIOS/ADIOS2IOHandler.cpp

```diff
@@ -1308,8 +1317,15 @@ void ADIOS2IOHandlerImpl::availableChunks(
     std::string varName = nameOfVariable(writable);
     auto engine = ba.getEngine(); // make sure that data are present
     auto datatype = detail::fromADIOS2Type(ba.m_IO.VariableType(varName));
+    bool allSteps = m_handler->m_frontendAccess != Access::READ_LINEAR &&
+        ba.streamStatus == detail::BufferedActions::StreamStatus::NoStream;
```

Comment:
Need to use `AllStepsBlocksInfo` if `allSteps` is true. This fixes https://github.com/openPMD/openPMD-viewer/issues/350.